molecule/default/molecule-no-systemd.yml

```diff
@@ -0,0 +1,78 @@
+---
+# This molecule configuration file is suitable for testing Ansible
+# roles that _do not_ require SystemD.  If your Ansible role _does_
+# require SystemD then you should use molecule-with-systemd.yml
+# instead.
+#
+# Note that the molecule configuration file that is symlinked to
+# molecule.yml is the one that will be used.
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint: |
+  set -e
+  yamllint .
+  ansible-lint
+  flake8
+platforms:
+  - name: amazonlinux2
+    image: amazonlinux:2
+  - name: debian10
+    image: debian:buster-slim
+  - name: debian11
+    image: debian:bullseye-slim
+  - name: kali
+    image: kalilinux/kali-rolling
+  - name: fedora32
+    image: fedora:32
+  - name: fedora33
+    image: fedora:33
+  - name: ubuntu16
+    image: ubuntu:xenial
+  - name: ubuntu18
+    image: ubuntu:bionic
+  - name: ubuntu20
+    image: ubuntu:focal
+provisioner:
+  name: ansible
+  config_options:
+    defaults:
+      # The OS family and distribution for Kali is "Kali GNU/Linux",
+      # which is not an allowable group name in Ansible due to the
+      # space and the slash.  This option converts those characters to
+      # underscores when creating the group name.
+      force_valid_group_names: always
+  inventory:
+    host_vars:
+      debian9:
```

Comment:
Since `debian9` is not a tested host do we still need this?

Reply:
I will remove it wholesale from the skeleton once we can drop Debian 9.  Until then I opted to keep it around just in case Debian 9 gets added again for some reason.